heron/tools/tracker/src/python/main.py

```diff
@@ -55,6 +55,8 @@ def __init__(self):
         (r"/topologies/executionstate", handlers.ExecutionStateHandler, {"tracker":self.tracker}),
```

Comment:
heron-ui is using it. If necessary I will refactor heron-ui in another PR. In the meantime, let us keep this API here.